packages/components/src/internal/components/user/UsersGridPanel.tsx

```diff
@@ -235,6 +217,9 @@ export class UsersGridPanelImpl extends PureComponent<Props, State> {
         if (model.selectionsLoadingState === LoadingState.LOADED) {
             this.updateSelectedUserId(this.getLastSelectedId());
         } else {
+            // TODO: This seems wrong, we should just do nothing, eventually the selections will load and we'll be able
+            //  to set the user id. If we don't automatically load selections we can manually call loadSelections via
+            //  model actions.
```

Comment:
yeah, I was looking at this component and the componentDidUpdate calls `setLastSelectedId` so I agree, wouldn't that catch this case and eventually just go into the if statement above?

Reply:
Yeah, it seems kinda wild that we would load the selections outside of `withQueryModels`, especially since it is very likely that it will load the selections anyway, so we'd end up with duplicate effort here. I'm not going to address this in this PR though, as it's already complicated enough.